### PR TITLE
Improved the content Flow.

### LINKED ISSUE #38603: Improve flow for Seccomp security tutorial
**This is a Feature Request**
Hi,
I would like to request a change made in seccomp.md doc page:
https://github.com/kubernetes/website/blob/b79539d0d35927eb51976d1115db500419a6c6b9/content/en/docs/tutorials/security/seccomp.md

It would make more sense if the "Create Pod that uses the container runtime default seccomp profile" section  was after "Enable the use of RuntimeDefault as the default seccomp profile for all workloads" section, and not the very last option of creating pods with seccomp profiles. I feel like it would improve the flow.

### PR DESCRIPTION
**Pull request by Shubham82**
This PR improved the content flow for the [Seccomp security tutorial](https://kubernetes.io/docs/tutorials/security/seccomp/).

Fixes #38603

### CODE REVIEW

**Comment by Shubham82** [comment]
Hi @hasheddan @pjbgf
PTAL!

File: content/en/docs/tutorials/security/seccomp.md [hunk 1]
```diff
@@ -265,6 +265,43 @@ docker exec -it kind-worker bash -c \
...
 ```
 
+## Create Pod that uses the container runtime default seccomp profile
+
+Most container runtimes provide a sane set of default syscalls that are allowed
+or not. You can adopt these defaults for your workload by setting the seccomp
+type in the security context of a pod or container to `RuntimeDefault`.
+
+{{< note >}}
+If you have the `SeccompDefault` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) enabled, then Pods use the `RuntimeDefault` seccomp profile whenever
```
Comment (tengqm) [inline content/en/docs/tutorials/security/seccomp.md:275]: nit:

```suggestion
If you have the `SeccompDefault` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
enabled, then Pods use the `RuntimeDefault` seccomp profile whenever
```

Comment (Shubham82) [reply content/en/docs/tutorials/security/seccomp.md:275]: Thank @tengqm for taking a look at this. I will update it.

Comment (Shubham82) [reply content/en/docs/tutorials/security/seccomp.md]: I have addressed the comment. PTAL!

**Review by saschagrunert — APPROVED** [review]

**Review by pjbgf — APPROVED** [review]
/lgtm

**Comment by Shubham82** [comment]
Hi @tengqm could you please approve this PR so that it will merge?

Thanks

**Pull request merged**